core/src/main/java/com/alipay/sofa/common/log/MultiAppLoggerSpaceManager.java

```diff
@@ -275,23 +276,23 @@ private static AbstractLoggerSpaceFactory createILoggerFactory(SpaceId spaceId,
                                   + SOFA_MIDDLEWARE_LOG_DISABLE_PROP_KEY + "=true");
             return NOP_LOGGER_FACTORY;
         }
-        // set log props
-        LogEnvUtils.processGlobalSystemLogProperties();
+
+        // set global system properties
+        spacesMap.get(spaceId).putAll(LogEnvUtils.processGlobalSystemLogProperties());
```

Comment:
spacesMap is `static final`, need use all uppercase letter.

Reply:
ok， fixed.